WebSocket handler fails to store response for debugging request - AccessDenied

## Steps to reproduce

- Attach plldb to any stack
- Invoke lambda in attached stack
- Check logs in websocket_default handler

## Expected behavior

The request is updated in PLLDBDebugger with the response code and response data

## Actual behavior

```
[ERROR] 2025-05-31T07:07:10.237Z 53818889-87da-46c4-a6d6-1bfec6edb656 Error updating DynamoDB: An error occurred (AccessDeniedException) when calling the UpdateItem operation: User: arn:aws:sts::xxx:assumed-role/plldb-PLLDBServiceRole-3yK81Ud1GtLx/plldb-websocket-default is not authorized to perform: dynamodb:UpdateItem on resource: arn:aws:dynamodb:eu-west-1:xxx:table/PLLDBDebugger because no identity-based policy allows the dynamodb:UpdateItem action
```

## Root cause

The PLLDBServiceRole does not have access privileges for DynamoDB

--- a/CLAUDE.md
+++ b/CLAUDE.md
@@ -7,6 +7,7 @@ This file provides guidance to Claude Code (claude.ai/code) when working with co
 - When fixing github issues, always create new branch for it, called issue/gh-<issue-id>
 - Before committing, run `make format` to format codebase
 - Pre-commit hooks are configured to run `make format`, `make pyright`, and `make test` automatically
+- Always link fixed GH issue into PR
 
 ## Commands
 

--- a/plldb/cloudformation/template.yaml
+++ b/plldb/cloudformation/template.yaml
@@ -110,6 +110,8 @@ Resources:
                 Resource:
                   - !GetAtt PLLDBSessions.Arn
                   - !Sub '${PLLDBSessions.Arn}/index/*'
+                  - !GetAtt PLLDBDebugger.Arn
+                  - !Sub '${PLLDBDebugger.Arn}/index/*'
               - Effect: Allow
                 Action:
                   - 'cloudformation:ListStackResources'

